CHANGELOG.md

```diff
@@ -31,6 +31,8 @@
 - PR #6768 Add support for scatter() on list columns
 - PR #6796 Add create_metadata_file in dask_cudf
 - PR #6765 Cupy fallback for __array_function__ and __array_ufunc__ for cudf.Series
+- PR #6805 Implement `cudf::detail::copy_if` for `decimal32` and `decimal64`
+- PR #6483 Add agg() function for dataframes
```

Comment:
```suggestion
- PR #6483 Add `agg` function for dataframes
```

Reply:
Can you make sure `CHANGELOG.md` has only a single line change that indicates the purpose of this PR only? Might be helpful to look at current latest code base: https://github.com/rapidsai/cudf/blob/branch-0.17/CHANGELOG.md